SPECS/python-cpuinfo/python-cpuinfo.spec

```diff
@@ -0,0 +1,220 @@
+%global srcname cpuinfo
+%global sum Getting CPU info
+
+Summary:        %{sum}
+Name:           python-%{srcname}
+Version:        7.0.0
+Release:        4%{?dist}
+License:        MIT
+URL:            https://github.com/workhorsy/py-cpuinfo
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        https://files.pythonhosted.org/packages/source/p/py-%{srcname}/py-%{srcname}-%{version}.tar.gz
+BuildArch:      noarch
+# https://github.com/workhorsy/py-cpuinfo/issues/55
+# ExclusiveArch:  %%{ix86} x86_64 %%{power64} s390x noarch
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-xml
+%if %{with_check}
+BuildRequires:  python3-pip
+%endif
+
+%description
+Py-cpuinfo gets CPU info with pure Python. Py-cpuinfo should work without
+any extra programs or libraries, beyond what your OS provides.
+
+These approaches are used for getting info:
+    Windows Registry
+    /proc/cpuinfo
+    sysctl
+    dmesg
+    isainfo and psrinfo
+    Querying x86 CPUID register
+
+%package -n python3-%{srcname}
+Summary:        %{sum}
+%{?python_provide:%python_provide python3-%{srcname}}
+
+%description -n python3-%{srcname}
+Py-cpuinfo gets CPU info with pure Python. Py-cpuinfo should work without
+any extra programs or libraries, beyond what your OS provides.
+
+These approaches are used for getting info:
+    Windows Registry
+    /proc/cpuinfo
+    sysctl
+    dmesg
+    isainfo and psrinfo
+    Querying x86 CPUID register
+
+%prep
+%setup -q -n py-%{srcname}-%{version}
+rm -rf *.egg-info
+
+sed -i -e '/^#!\//, 1d' cpuinfo/cpuinfo.py
+
+%build
+%py3_build
+
+%install
+%py3_install
+
+%check
+pip3 install atomicwrites>=1.3.0 \
+    attrs>=19.1.0 \
+    more-itertools>=7.0.0 \
+    pluggy>=0.11.0 \
+    pytest>=5.4.0 \
+    pytest-cov>=2.7.1
+PATH=%{buildroot}%{_bindir}:${PATH} \
+PYTHONPATH=%{buildroot}%{python3_sitelib} \
+    python%{python3_version} -m pytest -v tests
+
+%files -n python3-%{srcname}
+%license LICENSE
+%doc README.rst
+%doc ChangeLog
+%{_bindir}/cpuinfo
+%{python3_sitelib}/%{srcname}/
+%{python3_sitelib}/py_%{srcname}-%{version}-py3.*.egg-info
+
+%changelog
+* Wed Jun 23 2021 Rachel Menge <rachelmenge@microsoft.com> - 7.0.0-4
+- Update cgmanifest and license info
```

Comment:
>Update cgmanifest [](http://example.com/codeflow?start=2&length=17)

I'm not sure the cgmanifest updates need to be mentioned in this spec, since it's a different context. Unless the project/sources have their own cgmanifest, which you updated, but I don't think that's the case.